cmd/testing/pubsub/main.go

```diff
@@ -0,0 +1,136 @@
+// Pubsub is a simple manual test for ordered pubsub.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/birdie-ai/golibs/event"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// Event is the test event data.
+type Event struct {
+	PartitionID string
+	Count       int
+}
+
+func main() {
+	usage := func() {
+		log.Fatalf("usage: %q [subscriber|publisher]", os.Args[0])
+	}
+	if len(os.Args) <= 1 {
+		usage()
+	}
+	projectID := os.Getenv("GOOGLE_PROJECT")
+	if projectID == "" {
+		log.Fatal("missing env var: GOOGLE_PROJECT")
+	}
+	topicName := os.Getenv("TOPIC_NAME")
+	if topicName == "" {
+		log.Fatal("missing env var: TOPIC_NAME")
+	}
+
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt)
```

Comment:
as `<-ctx.Done()` is not selected, this prevents user from terminating the subscriber.